app/scripts/controllers/permissions/permissions-safe-methods.json

```diff
@@ -46,6 +46,8 @@
   "eth_submitWork",
   "eth_syncing",
   "eth_uninstallFilter",
+  "encryption_public_key",
+  "eth_decryptMessage",
```

Comment:
These methods should both probably use the `wallet_` prefix, as they are not eth-related, but they are related to the user's accounts, which is what we have used `wallet_` to mean.

Reply:
then we need to rewrite code to another npm package - https://github.com/MetaMask/eth-json-rpc-middleware/blob/master/wallet.js, but I see there 'personal_sign' as 'encryption_public_key ' and  'eth_signTypedData' as 'eth_decryptMessage'. Let's come to the single agreement for global naming and migrate to it. I think that need to do it simultaneously.